config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -92,9 +92,12 @@ IF(PYTHONINTERP_FOUND)
     # boost_python for the wrong Python version
     IF(Boost_FOUND)
         FIND_LIBRARY(Boost_PYTHON_LIBRARY
-                     NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}   # Linux with multiple Python versions
-                           boost_python-mt                                                 # Mac
-                           boost_python                                                    # default
+                     NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}      # Linux with multiple Python versions
+                           boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}-mt   # Linux with multiple Python versions (multithreaded)
+                           boost_python${PYTHON_VERSION_MAJOR}                                # Mac with Python 3
+                           boost_python${PYTHON_VERSION_MAJOR}-mt                             # Mac with Python 3 (multithreaded)
+                           boost_python                                                       # default
+                           boost_python-mt                                                    # Mac with Python (multithreaded)
```

Comment:
So it may not be doing any threading, but it is placing locks around any global state manipulations or using atomics.

Ok, we can place them first, but I still think the `Boost_USE_MULTITHREADED` variable should be respected.